Makefile.pkg

```diff
@@ -0,0 +1,66 @@
+# This Makefile contains RPM related targets
+COMMIT=$(shell git log --pretty=format:'%H' -n 1)
+COMMIT_DATE=$(shell git log --pretty='format:%cd' --date='format:%Y%m%d' -n 1)
+SHORT_COMMIT=$(shell git rev-parse --short=9 HEAD)
+MOCK_CONFIG=default
+ARCHIVE_BASE_NAME=avocado
+PYTHON_MODULE_NAME=avocado-framework
+RPM_BASE_NAME=python-avocado
+ifndef VERSION
+VERSION=$(shell $(PYTHON) setup.py --version 2>/dev/null)
+endif
+
+all:
+	@echo
+	@echo "RPM related targets:"
+	@echo "source: Create single source package with commit info, suitable for RPMs"
+	@echo "srpm:   Generate a source RPM package (.srpm)"
+	@echo "rpm:    Generate binary RPMs"
+	@echo
+	@echo "Release related targets:"
+	@echo "source-release:  Create source package for the latest tagged release"
+	@echo "srpm-release:    Generate a source RPM package (.srpm) for the latest tagged release"
+	@echo "rpm-release:        Generate binary RPMs for the latest tagged release"
+	@echo
+	@echo "variables:         Show the value of variables as defined in this Makefile or"
+	@echo "                   given as input to make"
+	@echo
+
+clean:
+	rm -rf SOURCES BUILD
+
+source: clean
+	if test ! -d SOURCES; then mkdir SOURCES; fi
```

Comment:
Just out of curiosity, in what cases this if is needed ? 

This target it will only be skipped if the previous return != 0 (in case of rm permission denied). However when the directory does not exist IMO it is safe to ignore and assume that the directory it was removed (that is the reason rm returns 0). 

So, maybe this if is not needed and you can create the directory anyway. Also mkdir has the `-p` that will ignore if it is already there.